src/HTTPure/Body.purs

```diff
@@ -8,17 +8,17 @@ module HTTPure.Body
 import Prelude
 
 import Data.Either as Either
+import Data.Foldable (foldMap)
+import Effect (Effect)
```

Comment:
Conventionally, in this project, I've gone with imports as qualified imports.  Maybe I'll change that, but if so, it will be after I put in some kind of automated style enforcer tool.  If you don't mind, please change to `import Data.Foldable as Foldable` and `import Effect as Effect` instead.